bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx120-na.html

```diff
@@ -0,0 +1,127 @@
+{#
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import picto with context %}
+
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('firefox_whatsnew_120_na') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+<section class="wnp-content-main">
+  <div class="mzp-l-content mzp-t-content-lg">
+    <h2 class="wnp-main-title">Make holiday shopping a breeze with Firefox</h2>
+    <div class="mzp-l-columns mzp-t-columns-three">
+       {% call picto(
+        title="See which reviews are real before buying your gifts",
+        heading_level=4,
+        class='box-one',
+        image=picture(
+          url='img/firefox/whatsnew/whatsnew120-na/light/headphones-light.svg',
+           sources=[
+            {
+              "media": "(prefers-color-scheme: light)",
+              "srcset": {
+                "img/firefox/whatsnew/whatsnew120-na/light/headphones-light.svg": "default",
+              }
+            },
+            {
+              "media": "(prefers-color-scheme: dark)",
+              "srcset": {
+                "img/firefox/whatsnew/whatsnew120-na/dark/headphones-dark.svg": "default",
+              }
+            }
+          ],
+          optional_attributes={
+            'class': 'mzp-c-picto-image',
+            'loading': 'lazy'
+          }
+        ),
+        body=True,
+      ) %}
+        <a class="mzp-c-button mzp-t-secondary"><img src="{{ static('img/firefox/whatsnew/whatsnew120-na/fakespot.svg')}}" width="24" height="24"><span>Install Fakespot</span></a>
+      {% endcall %}
+
+      {% call picto(
```

Comment:
**Nit (non-blocking):** There seems to be quite a bit of space above the CTAs, and it makes it difficult to distinguish which CTA belongs to which headline. I would suggest to decreasing the `margin-top` to `$spacing-md`, as it looks more visually balanced.
<kbd>![Screenshot 2023-11-20 at 3 27 15 PM](https://github.com/mozilla/bedrock/assets/42309026/afbf7580-12a9-450b-a689-92a0036bbb07)</kbd>